wiki/Help_Centre/Account_Restrictions/id.md

```diff
@@ -1,14 +1,11 @@
----
-outdated: true
----
-
 # Pembatasan Akses Akun
 
 Pembatasan akun adalah tindakan pembatasan dari komunitas yang diterapkan pada akun yang menunjukkan perilaku tidak wajar, mencurigakan, atau melanggar aturan.
 
 Ketika akses suatu akun dibatasi, akun tersebut tidak dapat berinteraksi dengan orang lain sampai sanksinya dicabut. Pemain yang dibatasi masih bisa memainkan game, mengunduh beatmap, dan mengirimkan skor, tetapi profil mereka tidak akan terlihat oleh orang lain.
 
 Fitur berikut dinonaktifkan untuk akun yang dibatasi:
+
 - *Chat* dalam game
```

Comment:
Bagian sini tolong diganti jadi percakapan  dalam permainan. Itu saran saya sih.
[en]: This part please replace to 'Percakapan dalam permainan'. That is my suggestion.